lib/matplotlib/widgets.py

```diff
@@ -403,12 +419,41 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt=None,
             valinit = valmin
         self.val = valinit
         self.valinit = valinit
+
+        marker_props = {}
+        defaults = {'facecolor':'white', 'edgecolor':'.75', 'size':10}
+        if handle_style is not None:
+            for k in ['facecolor', 'edgecolor', 'size']:
+                marker_props[f'marker{k}'] = handle_style.get(k, defaults[k])
+        else:
+            marker_props = {f'marker{k}': v for k, v in defaults.items()}
```

Comment:
You can simplify to:

```suggestion
        defaults = {'facecolor':'white', 'edgecolor':'.75', 'size':10}
        marker_props = {f'marker{k}': v for k, v in {**defaults, **handle_style}}
```
This has one desired functional difference: Invalid keys in `handle_style` will be promoted to `marker{key}` and thus raise when passed to plot. We don't want to accept arbitrary keys for `handle_style` and silently discard them.